engine/db_model_transactions.go

```diff
@@ -183,3 +179,15 @@ func (m *Transaction) migratePostgreSQL(client datastore.ClientInterface, tableN
 
 	return nil
 }
+
+func (m *Transaction) notify() {
+	if n := m.Client().Notifications(); n != nil {
+		n.Notify(notifications.NewRawEvent(&models.TransactionEvent{
```

Comment:
I'm not sure if it's possible in generics, but I would rather make n.Notify(...) to accept Event structs and convert them to RawEvents.

Reply:
There is a problem that methods cannot be generic in golang. 
- But I've introduced a utility genetic function which can simplify the job.
```golang
if n := m.Client().Notifications(); n != nil {
		notifications.Notify(n, &models.TransactionEvent{
                 // ......
       }
}
```